src/components/Connections/Connections.tsx

```diff
@@ -484,7 +487,14 @@ export const Connections = () => {
         ),
       ]);
 
+      const tempRowValues = data.map((connection: any) => [
+        connection.name, // as we are only sorting by connection name...
+        null,
+        null,
```

Comment:
Can we do it without the tempRowValues. Why not sort the orginal array and pass setRows in the List component to update.

Reply:
@mdshamoon The original array is an array of JSX which we cannot sort lexicographically, that's why a separate prop with just the **values** is given based on which the sorting happens

Reply:
Makes sense. I was hoping to achieve this with a single rows prop but this seems like the most logical approach.